explainer.md

```diff
@@ -0,0 +1,76 @@
+# Screen Capture Explained
+
+## What is it? What’s it for?
+
+It’s an API to get a live video stream of your computer screen or a single window on it, optionally with sound.
+
+It’s sometimes referred to it as screen-sharing, even though it’s only the capturing part being defined (sharing is just the primary use case).
+
+Have you ever been a presenter in an online video conference call? You probably shared your screen or the window of a presentation-program to do that. Have you ever presented in an online video conference call without this feature? That would probably require printouts and hand-waving! It’s a pretty key feature. This is that feature.
+
+
+ * Not the networking part—that’s WebRTC’s RTCPeerConnection
+ * Not the camera and microphone part—that’s MediaCapture’s getUserMedia
+ * **Just** the part that captures your computer's display. It’s called: getDisplayMedia
+
+You can capture the screen for other reasons as well, but this is the driving use-case.
+
+The approach is heavily modeled on our camera API, hence the name similarity. There’s a reason for that: both require user permission, and return a MediaStream object with a pair of highly privacy sensitive video and audio tracks—It’s just your screen instead of your face. The resulting MediaStream is a drop-in anywhere a MediaStream is accepted in the platform. Having obtained such a MediaStream, a website can trivially:
+
+ * play the live stream of your screen locally in a video element,
+ * send it over a peer connection to another client,
+ * send it to a MediaRecorder, perhaps for subsequent upload to a server,
+ * or all of the above (at once even).
+
+This flexibility is thanks to the MediaCapture-main spec which already did the heavy lifting of:
+ * defining MediaStream and MediaStreamTrack
+ * defining constraints an application can apply to downscale and decimate frame-rates
+ * defining a privacy-sensitive permission model
+
+This leaves this spec to focus on the parts unique to acquiring display media, and its privacy challenges.
+
+## Getting started
+
+This is how to request a video MediaStream sourced by screen-capture:
+
+```js
+const stream = await navigator.getDisplayMedia(video: true, audio: true);
+```
+
+Compare to a request for camera and microphone:
+
+```js
+const stream = await navigator.getUserMedia(video: true, audio: true);
+```
+These streams can then be used interchangeably:
+
+```js
+videoElement.srcObject = stream; // self-view
+
+new MediaRecorder(stream).start(); // record
+
+for (const track of stream.getTracks()) peerConnnection.addTrack(track, stream); // share
+
+await stream.getVideoTracks()[0].applyConstraints({height: 480}); // downscale
+
+for (const track of stream.getTracks()) track.stop(); // stop capture & relinquish access
+```
+
+Some important differences and unique properties of screen capture:
+
+ * The request is gated by end-user activation
```

Comment:
Check list construction.  You have a full sentence on the second-to-last bullet, which suggest that all bullets should be full sentences.